Cargo.toml

```diff
@@ -11,11 +11,18 @@ edition = "2018"
 rustc-hash = "1.0.1"
 text-size = "1.0.0"
 smol_str = "0.1.10"
-serde = { version = "1.0.89", optional = true, default-features = false }
 thin-dst = "1.0.0"
 
+[dependencies.serde_] # renamed so feature flag can be "serde"
+package = "serde"
+version = "1.0.89"
+optional = true
+default-features = false
+
 [dev-dependencies]
 m_lexer = "0.0.4"
 
 [features]
-serde1 = [ "serde", "text-size/serde" ]
```

Comment:
I named the feature `serde1` because, once, there was and idea that this'll work better with future `serde2`... Not sure what's the best practice here really

Reply:
The standard recommendation seems to be to [always just call the feature flag serde](https://github.com/rust-lang/api-guidelines/issues/180).

Reply:
See also the API guideline [C-feature](https://rust-lang.github.io/api-guidelines/naming.html#c-feature) which I recall mentioning preferring `serde` to `serde1` previously as well.